### PR TITLE
use g4 instead of p3

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: roles/openshift_node/tasks/create_machine.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 - name: Check machine is provisioned
   shell: >
-    oc get machines --kubeconfig={{ kubeconfig }} -n openshift-machine-api |grep p3.2xlarge |awk '{ print $2 }'
+    oc get machines --kubeconfig={{ kubeconfig }} -n openshift-machine-api |grep {{ gpu_instance_type }} |awk '{ print $2 }'
```
Comment (kpouget) [inline roles/openshift_node/tasks/create_machine.yml:12]: change `{{ gpu_instance_type` to `'{{ gpu_instance_type }}'` to avoid possible misuse/mistake

**Review by kpouget — COMMENTED** [review]
* the first patch hunk changes `s/p3.2xlarge/g4dn.xlarge` as per the title,
* the second patch hunk uses the `{{ gpu_instance_type }}` (referring to ^^^) instead of hard-coded value `p3.2xlarge`
  * the reference to the template should be protected to avoid any possible misuse/mistake

**Comment by sjug** [comment]
@ArangoGutierrez Lets combine this with https://github.com/openshift-psap/ci-artifacts/pull/17

**Pull request closed without merging**